### PR TITLE
Docblock: tribe_event_is_on_date()

### PR DESCRIPTION
**Pull request by deblynprado**

### CODE REVIEW

File: src/functions/template-tags/date.php [hunk 1]
```diff
@@ -56,13 +56,18 @@ function tribe_get_display_end_date( $event = null, $display_time = true, $date_
...
-	 * @param null $event
+	 * This function determines whether an event takes place on the specified date.
+	 * If no date is provided, the current date is used. If no event is provided,
+	 * the function will try to use the global post object.
 	 *
-	 * @return mixed|void
+	 * @since 3.10
+	 *
+	 * @param string|null $date The date to check against, in 'mysql' format. Defaults to current date.
+	 * @param WP_Post|null $event The event post object. Defaults to the global $post.
```
Comment (codingmusician) [inline src/functions/template-tags/date.php:68]: We need to line the `type`, `variable` and `description` up vertically.
```suggestion
	 * @param string|null  $date  The date to check against, in 'mysql' format. Defaults to current date.
	 * @param WP_Post|null $event The event post object. Defaults to the global $post.
```

**Comment by deblynprado** [comment]
I think it's good now @codingmusician 

Let me know if I'm still missing something 😅

**Comment by sdokus** [comment]
Don't forget to add the change to the `readme.txt` changelog (I forgot to go over that when I was showing you the process, sorry!)

**Review by codingmusician — APPROVED** [review]
Just needs a changelog (readme) and you're good to go!

**Comment by deblynprado** [comment]
@sdokus thought the changelog was just for the items we added the _deprecated_ tag :D 

Gonna get some example and add it

**Comment by deblynprado** [comment]
does this sounds good enough for the changelog?

> Improved the documentation for the `tribe_event_is_on_date()` function.

**Review by codingmusician — DISMISSED** [review]

**Review by bordoni — APPROVED** [review]

**Comment by bordoni** [comment]
Move to `release/T24.babayaga`

**Pull request closed without merging**